integration/storagetest/maptests.go

```diff
@@ -256,3 +256,45 @@ func (*mapTests) TestSetGetRoundTrip(ctx context.Context, t *testing.T, ms stora
 		}
 	}
 }
+
+func (*mapTests) TestLatestSignedMapRoot(ctx context.Context, t *testing.T, ms storage.MapStorage, as storage.AdminStorage) {
+	tree := mustCreateTree(ctx, t, as, storageto.MapTree)
+	mustSignAndStoreMapRoot(ctx, t, ms, tree, &types.MapRootV1{Revision: uint64(0)})
+	treeRev := int64(1) // After tree initialization, the first revision is 1.
+
+	timestamp := time.Now() // ensure the heads we create are newer than any initial head.
+	for i := int64(0); i < 10; i++ {
+		t.Logf("Starting iteration %d", i)
+		meta := []byte{byte(i)}
+
+		sth := &types.MapRootV1{
+			TimestampNanos: uint64(timestamp.Add(time.Duration(i) * time.Second).UnixNano()),
+			RootHash:       []byte(fmt.Sprintf("roothash %d", i)),
+			Revision:       uint64(treeRev + i),
+			Metadata:       meta,
+		}
+		mustSignAndStoreMapRoot(ctx, t, ms, tree, sth)
+
+		tx, err := ms.SnapshotForTree(ctx, tree)
+		if err != nil {
+			t.Errorf("%d: SnapshotForTree() = %v, want no error", i, err)
+			continue
+		}
+		got, err := tx.LatestSignedMapRoot(ctx)
+		if err != nil {
+			t.Fatalf("%d: LatestSignedMapRoot() = %v, want no error", i, err)
+		}
+
+		var gotSTH types.MapRootV1
+		if err := gotSTH.UnmarshalBinary(got.MapRoot); err != nil {
+			t.Fatalf("UnmarshalMapRootV1(): %v", err)
+		}
+
+		if !reflect.DeepEqual(&gotSTH, sth) {
+			t.Errorf("%d: LatestSignedMapRoot() \ngot: %#v \nwant:%#v", i, gotSTH, sth)
```

Comment:
```suggestion
			t.Errorf("%d: LatestSignedMapRoot()\ngot: %#v\nwant:%#v", i, gotSTH, sth)
```

Reply:
I like the bit of visual separation these spaces give at the end of a line - otherwise all the control codes kind of run together in my head. Acknowledging that these are only beneficial in code, not the output.